modules/installing-oci-about-agent-based-installer.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc [Using the Agent-based Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: CONCEPT
+[id="installing-oci-about-agent-based-installer_{context}"]
+= The Agent-based Installer and OCI overview
+
+You can install an {product-title} cluster on {oci-first} by using the Agent-based Installer Both Red Hat and Oracle test, validate, and support running {oci} and {ocvs-first} workloads in an {product-title} cluster on {oci}. 
```

Comment:
Add a `.` 

Example: ... by using the Agent-based Installer`.` Both Red Hat and Oracle test...